command/job_plan.go

```diff
@@ -153,13 +154,49 @@ func (c *JobPlanCommand) Run(args []string) int {
 		opts.PolicyOverride = true
 	}
 
+	if job.IsMultiregion() {
+		return c.multiregionPlan(client, job, opts, diff, verbose)
+	}
+
 	// Submit the job
 	resp, _, err := client.Jobs().PlanOpts(job, opts, nil)
 	if err != nil {
 		c.Ui.Error(fmt.Sprintf("Error during plan: %s", err))
 		return 255
 	}
 
+	exitCode := c.formatPlannedJob(job, resp, diff, verbose)
+	c.Ui.Output(c.Colorize().Color(formatJobModifyIndex(resp.JobModifyIndex, path)))
+	return exitCode
+}
+
+func (c *JobPlanCommand) multiregionPlan(client *api.Client, job *api.Job, opts *api.PlanOptions, diff, verbose bool) int {
+
+	var exitCode int
+
+	for _, region := range job.Multiregion.Regions {
+		regionName := region.Name
+		client.SetRegion(region.Name)
+
+		// Submit the job for this region
+		resp, _, err := client.Jobs().PlanOpts(job, opts, nil)
+		if err != nil {
+			c.Ui.Error(fmt.Sprintf("Error during plan for region %q: %s", regionName, err))
+			return 255
+		}
+
+		c.Ui.Output(c.Colorize().Color(fmt.Sprintf("[bold]Region: %q[reset]", regionName)))
+
+		regionExitCode := c.formatPlannedJob(job, resp, verbose, diff)
+		if regionExitCode > exitCode {
+			exitCode = regionExitCode
+		}
+	}
+	return exitCode
+}
+
+func (c *JobPlanCommand) formatPlannedJob(job *api.Job, resp *api.JobPlanResponse, diff, verbose bool) int {
```

Comment:
What do you think about `outputPlannedJob`?  I'm used to `format*` returning strings.

Reply:
Makes sense. Will do.